.vscode/launch.json

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
Is there much of a value in providing `launch.json`? I tend to avoid storing developers' IDE configs on the repo - just because you don't want to end up having configs for 10 different IDEs that people might use, and because devs may have different preferences and hence use whatever suits them best (which might not align with the configs on the repo). Also, it looks like it's autogenerated anyway, so I doubt there's a point in having it here.

Reply:
Yeah that's fair. This might be overkill